python_modules/libraries/dagster-looker/dagster_looker/api/resource.py

```diff
@@ -54,13 +64,141 @@ def read_config(_self) -> SettingsConfig:
 
         return init40(config_settings=DagsterLookerApiSettings())
 
+    @public
+    def build_defs(
+        self,
+        *,
+        request_start_pdt_builds: Optional[Sequence[RequestStartPdtBuild]] = None,
+        dagster_looker_translator: Optional[DagsterLookerApiTranslator] = None,
+    ) -> Definitions:
+        """Returns a Definitions object which will load structures from the Looker instance
+        and translate it into assets, using the provided translator.
+
+        Args:
+            request_start_pdt_builds (Optional[Sequence[RequestStartPdtBuild]]): A list of
+                requests to start PDT builds. See https://developers.looker.com/api/explorer/4.0/types/DerivedTable/RequestStartPdtBuild?sdk=py
+                for documentation on all available fields.
+            dagster_looker_translator (Optional[DagsterLookerApiTranslator]): The translator to
+                use to convert Looker structures into assets. Defaults to DagsterLookerApiTranslator.
+
+        Returns:
+            Definitions: A Definitions object which will build and return the Looker structures as assets.
```

Comment:
nit: "will build and return... as assets" -> "contains assets representing the Looker structures"

(since the Definitions object itself doesn't build)

Reply:
👍 thanks for catching